src/v2/Apps/Consign/OfferDetail/Components/SubmissionSummary.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box, Image, StackableBorderBox, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { get } from "v2/Utils/get"
+
+import { SubmissionSummary_offer } from "v2/__generated__/SubmissionSummary_offer.graphql"
+
+interface SubmissionSummaryProps {
+  offer: SubmissionSummary_offer
+}
+
+const SubmissionSummary: React.FC<SubmissionSummaryProps> = ({ offer }) => {
+  const { submission } = offer
+
+  return (
+    <StackableBorderBox>
+      <Box height="auto">{renderImage(submission)}</Box>
+      <Box>
+        <TruncatedLine>
+          <Text variant="mediumText">{submission.artist.name}</Text>
+          {renderTitleLine(submission)}
+        </TruncatedLine>
+        <TruncatedLine></TruncatedLine>
+      </Box>
+    </StackableBorderBox>
+  )
+}
+
+export const SubmissionSummaryFragmentContainer = createFragmentContainer(
+  SubmissionSummary,
+  {
+    offer: graphql`
+      fragment SubmissionSummary_offer on ConsignmentOffer {
+        submission {
+          artist {
+            name
+          }
+          title
+          year
+          assets {
+            imageUrls
+          }
+          primaryImage {
+            imageUrls
+          }
+        }
+      }
+    `,
+  }
+)
+
+function renderImage(submission: SubmissionSummary_offer["submission"]) {
+  const imageURL =
+    get(submission, s => (s.primaryImage.imageUrls as any).thumbnail) ||
```

Comment:
Any reason to use `get` over save access `?` operator?

Reply:
no! I just forgot about `?.`. I'll switch it!